tools/c7n_tencentcloud/tests/test_tc_cos.py

```diff
@@ -0,0 +1,106 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+from tc_common import BaseTest
+
+
+class TestCos(BaseTest):
+
+    @pytest.mark.vcr
+    def test_bucket_encryption(self):
+        policy = self.load_policy(
+            {
+                "name": "test_bucket_encryption",
+                "resource": "tencentcloud.cos",
+                "filters": [
+                    {"type": "bucket-encryption", "state": True, "crypto": "AES256"},
+                ]
+            }
+        )
+        resources = policy.run()
+        resources = [r for r in resources if r["Name"] == "custodian-test-1253831162"]
+        assert len(resources) == 1
+
+    @pytest.mark.vcr
+    def test_bucket_logging(self):
+        policy = self.load_policy(
+            {
+                "name": "test_bucket_logging",
+                "resource": "tencentcloud.cos",
+                "filters": [
+                    {
+                        "type": "bucket-logging",
+                        "op": "not-equal",
+                        "target_prefix": "{account_id}/{source_bucket_name}/",
+                        "target_bucket": "42342-1253831162"
+                    }
+                ]
+            }
+        )
+        resources = policy.run()
+        resources = [r for r in resources if r["Name"] == "custodian-test-1253831162"]
+        assert len(resources) == 1
+
+    @pytest.mark.vcr
+    def test_bucket_has_statement(self):
+        policy = self.load_policy(
+            {
+                "name": "test_has_statement",
+                "resource": "tencentcloud.cos",
+                "filters": [
+                    {
+                        "type": "has-statement",
+                        "statements": [
+                            {
+                                "Effect": "Allow",
+                                "Action": "name/cos:GetObject",
+                                "Principal": "qcs::cam::anyone:anyone"
+                            }
+                        ]
+                    }
+                ],
+            }
+        )
+        resources = policy.run()
+        resources = [r for r in resources if r["Name"] == "custodian-test-1253831162"]
```

Comment:
this can just be a filter in the policy itself

Reply:
done